vault/build.gradle

```diff
@@ -0,0 +1,50 @@
+plugins {
+    id 'war'
+    id 'maven'
+    id 'maven-publish'
+    id 'checkstyle'
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+apply from: '../opencadc.gradle'
+
+sourceCompatibility = 1.8
+
+group = 'org.opencadc'
+
+war {
+    from(System.getenv('RPS') + '/resources/') {
+        include 'swagger-ui/'
+    }
+    from('.') {
+        include 'VERSION'
+    }
+}
+
+dependencies {
+    compile 'org.opencadc:cadc-log:[1.1.6,2.0)'
+    compile 'org.opencadc:cadc-vosi:[1.4.3,2.0)'
+    compile 'org.opencadc:cadc-inventory-db:[0.14,1.0)'
```

Comment:
should not include cadc-inventory-db dependency because it's not needed (yet)

must not include postgresql at runtime - in fact should be in the runtime.exclude

is all that slf4j stuff cruft copied from someplace else or is it actually being pulled in? If not actually needed, please remove.

Reply:
cadc-inventory-db is used to get the SQLGenerator class name, used in vault.properties. But the SQLGenerator property isn't currently used.
Better to leave in the dependency, or pull it and that property?
Copied over another SI build.gradle to start with, can't remember which one. slf4j isn't pulled in. Removed slf4j from the build.

Reply:
remove usage and from build